.github/workflows/check-runtime-upgrade.yml

```diff
@@ -0,0 +1,136 @@
+name: Check runtime upgrade
+
+# Check runtime upgradability of the released runtime.
+# We do it in two ways:
+# - by executing try-runtime-cli against the live chain
+# - by simulating a runtime upgrade against the live chain
+
+on:
+  release:
+    types: [released]
+
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: runtime.wasm release_tag
+        required: false
+
+env:
+  SUBWASM_VERSION: 0.19.1
```

Comment:
the latest version is v0.21.3. Do we consider using that? Any dependency about polkadot version?

Reply:
Not in this PR

Reply:
We'll need to double-check if the new version has breaking change